corehq/apps/reports/standard/forms/reports.py

```diff
@@ -131,12 +131,22 @@ def _fmt_date(somedate):
             else:
                 form_name = EMPTY_FORM
                 form_username = EMPTY_USER
+
+            if xform_dict['doc_type'] == "XFormArchived":
+                archive_dates = [operation['date'] for operation in xform_dict['history']
+                                 if operation['operation'] == 'archive']
+                error_type = "{} ({})".format(
+                    SubmissionErrorType.display_name_by_doc_type(xform_dict['doc_type']),
+                    _fmt_date(string_to_utc_datetime(archive_dates[0])),
```

Comment:
i think this will fail with an obscure looking `IndexError` if there is no history, though I guess in theory that should be impossible...

Reply:
true. updated.